src/main/java/com/redhat/pantheon/servlet/ModuleListingServlet.java

```diff
@@ -52,14 +52,11 @@ protected String getQuery(SlingHttpServletRequest request) {
         // is not returning any results
         StringBuilder queryBuilder = new StringBuilder()
                 .append("select m.* from [nt:base] as m ")
-                    .append("INNER JOIN [pant:moduleVersion] as rev ON ISDESCENDANTNODE(rev, m) ")
+                    .append("INNER JOIN [pant:moduleLocale] as rev ON ISDESCENDANTNODE(rev, m) ")
```

Comment:
I think this might not ultimately solve the problem. Today we only have en_US locale nodes in the system, what about in the future where we have other locales? Would we get duplicate results at that point with this query?